config/jobs/image-pushing/k8s-staging-csi.yaml

```diff
@@ -0,0 +1,29 @@
+postsubmits:
+  # This is the github repo we'll build from. This block needs to be repeated
+  # for each repo.
+  kubernetes-csi/node-driver-registrar:
+    # The name should be changed to match the repo name above
+    - name: post-node-driver-registrar-push-images
+      cluster: test-infra-trusted
+      annotations:
+        # This is the name of some testgrid dashboard to report to.
+        testgrid-dashboards: sig-storage-csi-node-driver-registrar
+      decorate: true
+      branches:
+        - ^master$
```

Comment:
Can you add a comment explaining that this is for publishing the "canary" images, something that we only do on the master branch?

Reply:
Added comment